cluster/gce/gci/configure-helper.sh

```diff
@@ -676,9 +676,7 @@ function start-kube-addons {
     fi
     remove-salt-config-comments "${controller_yaml}"
     sed -i -e "s@{{ *base_metrics_memory *}}@${base_metrics_memory}@g" "${controller_yaml}"
-    sed -i -e "s@{{ *metrics_memory *}}@${metrics_memory}@g" "${controller_yaml}"
```

Comment:
Is it ok to remove these lines? I thought they were here to set the memory requirement for the heapster pods.